packages/lucia-auth/src/auth/index.ts

```diff
@@ -323,7 +323,7 @@ export class Auth<C extends Configurations = any> {
 
 type MaybePromise<T> = T | Promise<T>;
 
-interface Configurations {
+export interface Configurations {
```

Comment:
You have to re-export this in `src/index.ts`

Reply:
I pushed a commit which will hopefully fix this. I have been super busy and didn't get a chance to test it. I am not sure about your build configuration but the import in the `src/index.ts` file is `src/auth/index.js` and not `src/auth/index.ts` so I hope the types aren't removed by the stage of resolving the imports. Probably worth checking before a merge or I will check if I find some time.